docs/manual/cluster-admin/how-to-customize-alerts.md

```diff
@@ -0,0 +1,144 @@
+# How to Customize Alerts
+
+OpenPAI supports the customization of alert rules and corresponding handling actions.
+The alert rules are managed by `prometheus` service and the handling actions are managed by `alert-manager` service.
+
+In this document, we will introduce existing alerts & actions, their matching methods, and how to add new customized alerts & actions.
+
+## Existing Alerts/Actions & How to Match Them 
+
+### Existing Alerts
+
+OpenPAI uses `Prometheus` to monitor system metrics.
+We provide various alerts by defining rules on virtual_clusters, GPU utilization, etc.
+If OpenPAI is deployed, you can then visit `your_master_ip/prometheus/alerts` to see the details of alerts, including their definition and status.
+
+For example, `PAIJobGpuPercentLowerThan0_3For1h` alert rule is by default configured as :
+
+``` yaml
+alert: PAIJobGpuPercentLowerThan0_3For1h
+expr: avg(task_gpu_percent{virtual_cluster=~"default"}) BY (job_name) < 0.3
+for: 1h
+```
+
+It means that the job on virtual cluster `default` with task level average GPU percent lower than `30%` for more than `1 hour` will fire the alert `PAIJobGpuPercentLowerThan0_3For1h`.
+
+### Existing Actions
+
+OpenPAI uses `alert-manager` service for alert handling. So far, we have provided three types of actions: 
+
+* webportal-alert: Show alerts on the home page of webportal.
+* email-alert: Send email to the assigned receiver.
+* webhook-alert: Send alert via a POST request, we now support stopping jobs by adapting the POST request to the corresponding REST API of OpenPAI.
```

Comment:
The three types `webportal-alert`, `email-alert` and `webhook-alert` are classified according to the implementation, however, I don't think administrators care about the details. How about just listing the actions directly like
```markdown
* webportal-notification: Show alerts on the home page of webportal.
* email-notification: Send email to the assigned receiver.
* stopping-job: stopping corresponding job.
* tagging-job (TBA): add tags to corresponding job.
```

Reply:
> The three types `webportal-alert`, `email-alert` and `webhook-alert` are classified according to the implementation, however, I don't think administrators care about the details. How about just listing the actions directly like
> 
> ```gfm
> * webportal-notification: Show alerts on the home page of webportal.
> * email-notification: Send email to the assigned receiver.
> * stopping-job: stopping corresponding job.
> * tagging-job (TBA): add tags to corresponding job.
> ```

Renamed the action names and listed directly the actions.
Webhook-actions is remained here as an action type since stop-job and tag-job have some common configurations.